h2o-py/h2o/h2o.py

```diff
@@ -2394,38 +2344,38 @@ def estimate_cluster_mem(ncols, nrows, num_cols = 0, string_cols = 0, cat_cols =
     >>> ### because I know 4 of 8 columns are categorical and 4 of 8 columns consist of numbers.
     >>> estimate_cluster_mem(ncols=8, nrows=31000000, cat_cols=4, num_cols=4)
     
-    """    
+    """
     import math
-    
-    if (ncols < 0):
+
+    if ncols < 0:
         raise ValueError("ncols can't be a negative number")
-    
-    if (nrows < 0):
+
+    if nrows < 0:
         raise ValueError("nrows can't be a negative number")
-    
-    if (num_cols < 0):
+
+    if num_cols < 0:
         raise ValueError("num_cols can't be a negative number")
-    
-    if (string_cols < 0):
+
+    if string_cols < 0:
         raise ValueError("string_cols can't be a negative number")
-    
-    if (cat_cols < 0):
+
+    if cat_cols < 0:
         raise ValueError("cat_cols can't be a negative number")
-    
-    if (time_cols < 0):
+
+    if time_cols < 0:
         raise ValueError("time_cols can't be a negative number")
-    
-    if (uuid_cols < 0):
+
+    if uuid_cols < 0:
         raise ValueError("uuid_cols can't be a negative number")
-    
-    BASE_MEM_REQUIREMENT_MB = 32
-    SAFETY_FACTOR = 4
-    BYTES_IN_MB = 1024 * 1024
-    BYTES_IN_GB = 1024 * BYTES_IN_MB
+
+    base_mem_requirement_mb = 32
```

Comment:
Why do you rename these constants? It looks like constants, or am I missing something?

Reply:
If those were defined at module level, or even class level, these would be constants and could be all upper-case and respect the Py coding style (https://www.python.org/dev/peps/pep-0008/#constants), but they are defined inside a function and as such, they're just variables, so they don't follow the PEP, and IDEs like IntelliJ keep complaining/underlining, creating visual noise and this works pretty well because it's annoying me and I fix it to respect the PEP :)

Reply:
Oh, I did not check if they are in the function! So then you are absolutely correct. :)